django/contrib/admin/sites.py

```diff
@@ -443,7 +443,7 @@ def _build_app_dict(self, request, label=None):
                 'object_name': model._meta.object_name,
                 'perms': perms,
             }
-            if perms.get('change'):
+            if perms.get('change', False) or perms.get('view', False):
```

Comment:
No need to use `False` as a default value here, `None` also evaluates to `False`.